pkg/ctl/cmdutils/nodegroup_flags.go

```diff
@@ -37,6 +37,7 @@ func AddCommonCreateNodeGroupFlags(fs *pflag.FlagSet, cmd *Cmd, ng *api.NodeGrou
 
 	ng.SSH.Allow = fs.Bool("ssh-access", *ng.SSH.Allow, "control SSH access for nodes. Uses ~/.ssh/id_rsa.pub as default key path if enabled")
 	ng.SSH.PublicKeyPath = fs.String("ssh-public-key", "", "SSH public key to use for nodes (import from local path, or use existing EC2 key pair)")
+	ng.SSH.EnableSSM = fs.Bool("enable-ssm", false, "Enable simple systems manager (SSM)")
```

Comment:
According to [this doc](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html), it was formerly known as Amazon Simple Systems Manager (SSM), now it's just AWS Systems Manager.

```suggestion
	ng.SSH.EnableSSM = fs.Bool("enable-ssm", false, "Enable AWS Systems Manager (SSM)")
```

Reply:
good spot!